content/workshops/intro-to-css-architecture/_index.md

```diff
@@ -1,4 +1,16 @@
 ---
 title: Intro to CSS architecture
+ready: true
 ---
 
+The followings are to be demonstrated and addressed during the workshop
+
+1. Why the need to organize our CSS
+2. Separation of Concerns (what is it ?)
+3. Modular Architecture
+
+## Resources
+
+1. [CSS Architecture and the Three Pillars of Maintainable CSS](https://www.sitepoint.com/css-architecture-and-the-three-pillars-of-maintainable-css/)
```

Comment:
This isn't DRY. It's literally copy-pasted across different markdown files.  Treat department documentation like code. It must be DRY and cohesive. 

If you want to refer to a piece of content from another pieve of content then you can use the `contentlink` shortcode. Eg:

  ` {{% contentlink "topics/intro-to-css" %}}`

Please always be DRY. Always aim to be DRY. It will make you a better developer if you do DRY things